js/components/CreateItemNav/styles.js

```diff
@@ -0,0 +1,55 @@
+import { StyleSheet } from 'react-native';
+import { fonts, colors } from '../../config/styles';
+
+const styles = StyleSheet.create({
+  icon: {
+    height: 45,
+    width: 45,
+    padding: 7,
+    marginBottom: 10,
+  },
+  headerOption: {
+    width: 30,
+    height: 30,
+    margin: 10,
+  },
+  container: {
+    backgroundColor: 'white',
+    borderColor: colors.darkBlue,
+    borderWidth: 1,
+    borderRadius: 7,
+    position: 'absolute',
+    bottom: 0,
+    right: '40%',
+    alignContent: 'center',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    width: '100%',
+  },
+  option: {
+    borderColor: colors.darkBlue,
+    borderBottomWidth: 1,
+    width: '100%',
+  },
+  cancel: {
+    color: colors.darkBlue,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    width: '100%',
+  },
+  optionText: {
+    color: colors.darkBlue,
+    fontFamily: fonts.main,
+    fontWeight: 'bold',
+    fontSize: 18,
+    padding: 5,
+    paddingLeft: 5,
+    paddingRight: 5,
```

Comment:
If you have padding: 5, you are over righting that with paddingLeft and paddingRight. You can improve your styles by using flex box.